apps/experiments/versioning.py

```diff
@@ -125,14 +130,19 @@ def _compare_queryset(self, previous_queryset):
                 # TODO: When comparing static trigger versions and only the action changed, it is not being picked up.
                 previous_record_version_ids.append(previous_record.id)
                 prev_version_field = VersionField(raw_value=previous_record, to_display=self.to_display)
-                version_field.compare(prev_version_field)
+                version_field.compare(prev_version_field, early_abort=early_abort)
                 self.changed = self.changed or version_field.changed
             else:
                 version_field.changed = self.changed = True
 
+        if early_abort and self.changed:
```

Comment:
this could be in the for loop which would abort even earlier if a change was detected

Reply:
Updated